.github/workflows/download-data.yml

```diff
@@ -0,0 +1,44 @@
+name: download-data
+
+on:
+    schedule:
+        # Run everyday at 2:00pm CST which is 7:00pm UTC
+        - cron: 0 19 * * *
+
+    # Allows the workflow to be run manually from the Actions tab
+    workflow_dispatch:
+
+    push:
+        branches:
+            - github-action
+
+
+jobs:
+    download-data-json:
+        runs-on: macos-latest
```

Comment:
May I know the context around the choice for running on macos instead of ubuntu?

Reply:
`macos` runners have 14 GB of RAM, but `ubuntu` only has 7. [https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners). This was an issue for one of the backend jobs, but `ubuntu` might be fine for this one.

Reply:
I have no concerns with `macos`. I was just concerned about whether we were running up against some memory limits that we might some day exceed again.